cps/tasks/download.py

```diff
@@ -80,9 +80,8 @@ def run(self, worker_thread):
                                     last_progress_time = datetime.now()
                     else:
                         elapsed_time = (datetime.now() - last_progress_time).total_seconds()
-                        if elapsed_time >= fragment_stuck_timeout:
-                            self.message += f"<br>Some fragments are taking longer than expected to download. Please wait..."
-
+                        if elapsed_time >= timeout:
+                            self.message = f"{self.media_url_link} is stuck due to unavailable fragments. See <a href='https://github.com/yt-dlp/yt-dlp/issues/2137' target='_blank'>yt-dlp/yt-dlp#2137</a> for context."
```

Comment:
```suggestion
                            self.message = f"{self.media_url_link} is taking longer than expected. It could be a stuck download due to unavailable fragments (https://github.com/yt-dlp/yt-dlp/issues/2137) and/or an error in xklb’s media check. (See (<a href=‘https://github.com/iiab/calibre-web/pull/223' target=‘_blank’>https://github.com/iiab/calibre-web/pull/223 for more info."
```

Reply:
This is better.